.gitignore

```diff
@@ -20,4 +20,5 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
+.env
```

Comment:
It's preferred to keep Pull Request tied to one specific issue. In this case it's fine because it's such a minor change, but in the future a PR should do one thing